docs/run-openmetadata-on-kind-k8s.md

```diff
@@ -0,0 +1,25 @@
+# Running OpenMetadata on Kind Kubernetes
+
+These instructions are based on: https://github.com/open-metadata/OpenMetadata/issues/6324
+
+The instructions explain how to install OpenMetadata in the `default` namespace. Make sure that the current namespace is `default`:
+
+```bash
+kubectl config set-context --current --namespace=default
+```
+
+As the instructions indicate, one needs to run:
+
+```bash
+kubectl apply -f pv1.yaml
```

Comment:
what are pv1.yaml and pv2.yaml, where should I take them? Should I clone one of OpenMetadata repos ?

Reply:
copied to the `third_party/openmetadata` directory